### PR TITLE
Enable Xdebug

### PR DESCRIPTION
**Pull request by Firesphere**
This enables Xdebug. Included a description for how to get Xdebug working on PHPStorm/IntelliJ in the comments of the script.
As followup to issue #58

### CODE REVIEW

File: scripts/xdebug.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+echo "Install Xdebug"
+yum install -y gcc gcc-c++ autoconf automake --enablerepo=epel --enablerepo=webtatic
+
+pecl install Xdebug
+
+echo "[xdebug]
+zend_extension=\"/usr/lib64/php/modules/xdebug.so\"
+xdebug.remote_connect_back = 1
+xdebug.remote_enable = 1" >> /etc/php.ini
```
Comment (dhensby) [inline scripts/xdebug.sh]: can't we echo this into it's own .ini that gets auto included? to save this being appended repeatedly if the script is run more than once?

**Comment by willmorgan** [comment]
What Dan said + squashing = merge, I think. Thanks for doing this dude.

**Comment by Firesphere** [comment]
Heh, valid point about the single ini. My php.ini apparently has 4 xdebug lines. Will try to get this done today or tomorrow.

**Comment by Firesphere** [comment]
I seem to be an idiot with squashing, re-commiting everything. I'll create a clean patch for this and close this pull request, before I go crazy.

**Pull request closed without merging**